README.md

```diff
@@ -3,15 +3,109 @@
 Ralph is a toolbox for your learning analytics, it can be used as a:
 
 - **library**, to fetch learning events from various backends, (de)serialize or
-  convert them from various standard formats such as
-  [xAPI](https://adlnet.gov/projects/xapi/),
+    convert them from various standard formats such as
+    [xAPI](https://adlnet.gov/projects/xapi/), or
+    [openedx](https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_logs/index.html)
 - **command-line interface** (CLI), to build data pipelines the UNIX-way™️,
 - **HTTP API server**, to collect xAPI statements (learning events)
   following the [ADL LRS
   standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree).
 
 ## Quick start guide
 
+### Testing the LRS server with Docker compose
```

Comment:
I think a small prerequisite for docker compose would be nice, WDYT? On what version is needed and maybe a link to know how to install it?

Reply:
Good point. I've added preliminary notes for this.